frontend/src/scenes/billing/Billing.tsx

```diff
@@ -152,144 +160,28 @@ export function Billing(): JSX.Element {
-                                                        </LemonLabel>
-                                                        <div className="font-semibold text-2xl text-secondary">
-                                                            {humanFriendlyCurrency(billing?.discount_amount_usd, 0)}
-                                                        </div>
-                                                    </div>
-                                                )}
-                                                {billing?.discount_percent && (
-                                                    <div>
-                                                        <LemonLabel
-                                                            info="The discount applied to your current bill, reflected in the total amount."
-                                                            className="text-secondary"
-                                                        >
-                                                            Applied discount
-                                                        </LemonLabel>
-                                                        <div className="font-semibold text-2xl text-secondary">
-                                                            {billing.discount_percent}%
-                                                        </div>
-                                                    </div>
-                                                )}
-                                            </div>
-                                        </>
-                                    )}
-                                    <div>
-                                        <p className="ml-0 mb-0">
-                                            {billing?.has_active_subscription ? 'Billing period' : 'Cycle'}:{' '}
-                                            <b>{billing.billing_period.current_period_start.format('LL')}</b> to{' '}
-                                            <b>{billing.billing_period.current_period_end.format('LL')}</b> (
-                                            {billing.billing_period.current_period_end.diff(dayjs(), 'days')} days
-                                            remaining)
-                                        </p>
-                                        {!billing.has_active_subscription && (
-                                            <p className="italic ml-0 text-secondary mb-0">
-                                                Monthly free allocation resets at the end of the cycle.
-                                            </p>
-                                        )}
-                                    </div>
-                                </div>
-                            </div>
-                        )}
+                {!!billingPlan && !billing?.trial && !!platformAndSupportProduct && (
+                    <div className="flex-1">
```

Comment:
Mobile looks good but it doesn't seem right on deskop. Could we flip so the summary is on the left? Trying to imagine how this could best fit.

Reply:
Yeah I think you're right, this does look better, wdyt?

![image](https://github.com/user-attachments/assets/77bfc64c-e92a-4b96-b0cb-ed50d39135d6)

Reply:
Yeah, and removing some of the paragraph copy will make it shorter and look better, as well.

Reply:
Yup true

![image](https://github.com/user-attachments/assets/f348e5a6-6469-48bb-af26-d2e8ce8d6955)